docs/overview/introduction.mdx

```diff
@@ -18,8 +18,10 @@ Alerts usually categorized into three different groups:
 Keep helps with every step of the alert lifecycle:
 1. Creation - Keep offers a framework for creating, debugging, and testing alerts through code that scales with your teams.
 2. Maintenance - Keep integrates with your tools, allowing you to manage all of your alerts within a single interface.
-3. Noise reduction - By integrating with monitoring tools, Keep can deduplicate and correlate alerts to reduce noise in your organization.
+3. Noise reduction - By integrating with monitoring tools, Keep can deduplicate and correlate alerts to reduce noise in your organization. Rule-based (all distributions), and AI-based (Keep Enterprise only).
 4. Automation - [Keep Workflows](/workflows) enable automated alert enrichment and response.
+5. Incident Correlation - Automatically assign alerts to incidents (Keep Enterprise only).
```

Comment:
Explain who this differs from #2?

Reply:
Different things people will search for I guess, that's why it worth mentioning them both